doc/PLUGINS.md

```diff
@@ -777,7 +777,7 @@ no plugin is registered on the hook.
 ### `peer_connected`
 
 This hook is called whenever a peer has connected and successfully completed
-the cryptographic handshake. The parameters have the following structure if there is a channel with the peer:
+the cryptographic handshake. The parameters have the following structure:
```

Comment:
This seems unrelated.

Reply:
When reading the doc and reading the code that produces this DTO for the hook, the part "if there is a channel with the peer" seemed wrong to me, as the parameters will always be the same regardless if there is a channel or not. This is why I removed this part.